elevate-cpanel

```diff
@@ -173,6 +173,12 @@ use Cpanel::Version::Tiny       ();
 use Cpanel::Version::Compare    ();
 use Cpanel::Yum::Vars           ();
 
+#<<V *** DO NOT EDIT THESE LINES MANUALLY ***
+use constant VERSION => 0;
```

Comment:
Why a constant for $VERSION?

Reply:
Does this not resolve to the same thing as `sub VERSION { ... }`? Or would you prefer the `our $VERSION = ...` convention?